### DISCUSSION TITLE
More freedom in editing

### DISCUSSION THREAD

**Discussion by Lachrimae93** [JohanFalt/Foundry_WoD20#279] [Ideas]
Hi. Great Sheet. i just wanted to suggest if you could allow people to edit more of the sheet (such as inputting a path because you didn't include all of them in the dropdown or adding specialties to anywhere because some skills allow specialties to be added even when you arne't at 4 dots yet but the sheet doesn't account for that)? There's just so little freedom with how things are edited that it can be cumbersome to customize.

**Comment by JohanFalt** [comment]
This will be fixed in the coming verison (1.7), details here:

https://github.com/JohanFalt/Foundry_WoD20/issues/161

**Comment by Lachrimae93** [comment]
Was this closed because it can't be done? Because it seems the ability to edit the paths still doesn't exist

**Comment by JohanFalt** [comment]
Paths are scheduled for v2.5.
